README.md

```diff
@@ -34,6 +34,13 @@ Supported options along with their defaults are shown below.
 custom:
   slicWatch:
     topic: SNS_TOPIC_ARN
+    dashboard:
+      timeRange:
+        start: '-PT3H'
```

Comment:
do we need also an `end`?

Reply:
Not for the defaults, since end is optional and usually unused.